fluentui_persona/src/main/java/com/microsoft/fluentui/tokenised/persona/Avatar.kt

```diff
@@ -0,0 +1,222 @@
+                if (enablePresence)
+                    Image(image.value(themeMode), "Status", Modifier.offset(presenceOffset.x, presenceOffset.y))
+            }
+        }
+    }
+}
+
+@Composable
+fun Avatar(
+        group: Group,
+        size: AvatarSize = AvatarSize.Medium,
+        imageNAStyle: AvatarImageNA = AvatarImageNA.Standard,
+        modifier: Modifier = Modifier,
+        avatarToken: AvatarTokens? = null,
+) {
+
+    val token = avatarToken
+            ?: FluentTheme.controlTokens.tokens[ControlTokens.ControlType.Avatar] as AvatarTokens
+
+    CompositionLocalProvider(
+            LocalAvatarTokens provides token,
+            LocalAvatarInfo provides AvatarInfo(size, imageNAStyle, false)
+    ) {
+        val avatarSize = getAvatarTokens().avatarSize(getAvatarInfo())
+        val bordersRadius = getAvatarTokens().borderRadius(getAvatarInfo())
+        val fontSize = getAvatarTokens().fontSize(getAvatarInfo())
+
+
+        Box(modifier.requiredSize(avatarSize), contentAlignment = Alignment.Center
+        ) {
+            Box(Modifier
+                    .clip(RoundedCornerShape(bordersRadius))
+                    .background(getAvatarTokens().backgroundColor(getAvatarInfo()))
+                    .fillMaxSize(),
+                    contentAlignment = Alignment.Center
+            ) {
+                if (group.image != null) {
+                    Image(
+                            painter = painterResource(group.image),
+                            contentDescription = group.groupName,
+                            modifier = Modifier
+                                    .size(avatarSize)
+                                    .clip(RoundedCornerShape(bordersRadius))
+                    )
+                } else if (group.imageBitmap != null) {
+                    Image(
+                            bitmap = group.imageBitmap,
+                            contentDescription = group.groupName,
+                            modifier = Modifier
+                                    .size(avatarSize)
+                                    .clip(RoundedCornerShape(bordersRadius))
+                    )
+                } else if (imageNAStyle == AvatarImageNA.Initials) {
+                    Text(group.getInitials(), fontSize = with(LocalDensity.current) { fontSize.toSp() },
+                            color = Color(0xFF6E0911))
+                } else {
+                    Icon(
+                            getAvatarTokens().icon(getAvatarInfo()),
+                            "${group.groupName} Anonymous",
+                            tint = getAvatarTokens().iconColor(getAvatarInfo())
+                    )
+                }
+            }
+        }
+    }
+}
+
+@Composable
+fun Avatar(overflowCount: Int,
+           size: AvatarSize = AvatarSize.Medium,
+           modifier: Modifier = Modifier,
+           avatarToken: AvatarTokens? = null,
```

Comment:
shift avatarToken to be the last parameter, which give clean way of creating anonymous class

Reply:
Keep parity among function parameters across all the functions

Reply:
Similar Structure was followed in button. Please clarify on requirements.

Reply:
We need to update their as well when package name get updated.